xs/src/libslic3r/ConfigBase.hpp

```diff
@@ -1,6 +1,10 @@
 #ifndef slic3r_ConfigBase_hpp_
 #define slic3r_ConfigBase_hpp_
 
+#include <execinfo.h>
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
What is getting used here that is in stdlib? If it's actually getting used, should be `<cstdlib>`